src/ReactiveUI/Cocoa/CommonReactiveSource.cs

```diff
@@ -330,7 +330,19 @@ private void SubscribeToSectionInfoChanges(int sectionInfoId, IReadOnlyList<TSec
                                             });
                                     }
 
-                                    this.pendingChanges.Add(Tuple.Create(y.Section, new PendingChange(y.Change)));
```

Comment:
The related issue describes this problem as being to do with index normalization. Did that not turn out to be the case then? If not, should the responsibility be moved there? Is this a platform-specific thing, or iOS only?